### ISSUE TITLE
Accessibility: render header="......" as <h2> (p-panel and p-dialog)

### ISSUE THREAD

**Issue by moma1806** [primefaces/primeng#6056]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => Search github for a similar issue or PR before submittin

```
**Current behavior**
the attribute p-panel header="Godfather I" and p-dialog header="Title" render the title as a span-element.

**Expected behavior**
for good accessibility, headers should be delivererd as h1-h6 tags.
Headings communicate the organization of the content on the page. Web browsers, plug-ins, and assistive technologies can use them to provide in-page navigation.
https://www.w3.org/WAI/tutorials/page-structure/headings/

**Minimal reproduction of the problem with instructions**
Example on showcase
https://www.primefaces.org/primeng/#/panel
https://www.primefaces.org/primeng/#/dialog

**What is the motivation / use case for changing the behavior?**
Now we use the  component p-header with h2 inside
but using the attribute header=".." is more efficient and avoids inconsitency, when both header="Godfather I" and are p-header used in different parts of the code.

Please tell us about your environment:
Windows 10, Eclipse Oxygen 4.7

    Angular version: 5.X
    version: 5.1.0

    PrimeNG version: 5.X
    version: 5.2.0

    Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
    all

    Language: [all | TypeScript X.X | ES6/7 | ES5]
    all

    Node (for AoT issues): node --version =
    version: 7.5.0

**Comment by cagataycivici** [comment]
It will break styling of existing apps, so unable to do it.
